src/main/java/core/basesyntax/model/User.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class User {
```

Comment:
here as well - add all getters and setters

Reply:
done